scripts/bundled_installer

```diff
@@ -31,14 +34,21 @@ function exit_if_return_code_is_non_zero() {
     fi
 }
 
+function echo_recommendation_message() {
+    local message=$1
+    echo_with_indentation "\033[38;5;11m$message\033[0m"
+}
+
 function install_python() {
     echo ""
+    SHOW_PATH_EXPORT_MESSAGE=false
```

Comment:
You do not need this until you get to the `if` block below. Move this variable definition inside the `if` block.